GameFiles/_integration.gsc

```diff
@@ -38,13 +34,35 @@ init()
 
     level.clientCommandCallbacks = [];
     level.clientCommandRusAsTarget = [];
+    level.overrideMethods = [];
+
+    wait 0.1;
```

Comment:
What's the purpose of this wait? It's not immediately clear so I would add a comment of why we're doing this.

Reply:
That is just some leftover, removed with 7510ebd